recipes/sigmut/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+./sigprofiler -ig GRCh38
```

Comment:
I’ve tested this locally and it appears that this script will download 3 GB of data (the human reference genome). Is the intention to include that in the final package? That would result in a very big package and I don’t think that’s appropriate.

Do you want to test the script? Then you should add something like this to the `test:` commands.

Reply:
You're completely right @marcelm. I will change the file according to your suggestions and make the PR.